jujugui/static/gui/src/app/components/user-profile/entity/test-entity.js

```diff
@@ -321,7 +333,7 @@ describe('UserProfileEntity', () => {
     });
   });
 
-  it('can toggle to the expanded view', () => {
+  xit('can toggle to the expanded view', () => {
```

Comment:
Looking at the react source (and I suppose we should have assumed this by the method name) you can't find a DOM node if there is no DOM node created :) 

Are you able to use a ref in the setState instead of calling `findDOMNode` ?